app/components/service/includes/resources.html

```diff
@@ -6,31 +6,41 @@
       <table class="table">
         <tbody>
           <tr>
-            <td>CPU limits</td>
-            <td ng-if="service.LimitNanoCPUs">
-              {{ service.LimitNanoCPUs / 1000000000 }}
+            <td>CPU limit<portainer-tooltip position="top" message="Maximum number of CPU the service can use (0 to unlimited)."></portainer-tooltip></td>
```

Comment:
Please replace the tooltip message with: `Maximum CPU usage per task (set to 0 for unlimited)`